tests/scores/categorical/test_multicategorical.py

```diff
@@ -253,6 +253,30 @@ def test_firm(
     )
 
 
+def test_firm_dask():
+    """Tests firm works with dask"""
+    calculated = firm(
+        mtd.DA_FCST_FIRM.chunk(),
+        mtd.DA_OBS_FIRM.chunk(),
+        0.7,
+        [0, 5],
+        mtd.LIST_WEIGHTS_FIRM2,
+        0,
+        reduce_dims=None,
+        preserve_dims=["i", "j", "k"],
+    )
+
+    calculated = calculated.transpose(*["i", "j", "k"])
+    assert len(calculated.chunks) == 3
+    calculated = calculated.compute()
+    assert len(calculated.chunks) == 0
```

Comment:
A safer way to check if the underlying data is still a dask array would be to call:
`isinstance(calculated.data, dask.array.Array)`
or more duck-typing way:
`hasattr(calculated.data, "dask")`

That way you aren't relying on the exact number of chunks.

Reply:
`isinstance(calculated.data, dask.array.Array)` is much nicer thanks